.env.template

```diff
@@ -4,4 +4,5 @@ VITE_REMOTE_HEADER_MENU_ID=23
 VITE_REMOTE_FOOTER_MENU_ID=24
 VITE_REMOTE_SOCIAL_LINKS_ID=26
 VITE_LOCATION_API_URL=https://www.ucf.edu/wp-json/wp/v2/locations/
-VITE_BASEURL= /map/
+VITE_BASEURL=/map/
+VITE_REACTGA_MEASUREMENT_ID=G-DFCVBKX579
```

Comment:
We probably need to set this up in such a way that it can be run without analytics, so that when we're just running it locally it's not reporting data. In addition, we definitely don't want our DEV/TEST ID to be the default thing people are using, because we'd be getting data reported from folks running this locally. This default value should be replaced with some sort of template value, or an empty string of some sort with instructions to set it if we're using it.

Reply:
It was changed to empty string